.github/workflows/main.yaml

```diff
@@ -0,0 +1,160 @@
+name:  Release Main
+env:
+  ARGO_NAMESPACE: argo
+  ARGO_VERSION: v3.1.11
+# this needs to be a template
+  GITREPO_URL: ssh://git@<GITHUB_HOST>/<GITHUB_OWNER>/metaphor.git
+  CI_PROJECT_NAME: metaphor
```

Comment:
Github variables don't work at this level, this were kept like this to make easier to read the code and simplify use on the other metaphors repo.